tests/features/support/steps_clone.js

```diff
@@ -17,173 +18,173 @@ KEY:
 // let ordinal_to_integer = {
-//   first: 0, second: 1, third: 2, fourth: 3, fifth: 4,
+//   first: 0, second: 1, third: 2, fourth: 3, fifth: 4, 
 //   sixth: 5, seventh: 6, eighth: 7, ninth: 8, tenth: 9,
 //   '1st': 0, '2nd': 1, '3rd': 2, '4th': 3, '5th': 4,
 //   '6th': 5, '7th': 6, '8th': 7, '9th': 8,'10th': 9,
 // };
 // let specified = '?"?(first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth|"[^"]+)?"?';
-// let click_with = { mobile: 'tap', pc: 'click', };
+// let click_with = { mobile: 'tap', pc: 'click', }; 
 // // non-capturing capturing group expression for single or double quotes?
 
 
 // =========================
 // Establishing
 // =========================
-//x @step(r'I spend at least ([0-9]+) seconds? on each page')
+//x @step(r'I spend at least ([0-9]+) seconds? on each page') 
 //x @step(r'I want to store screenshots in the folder "([^"]+)"')
 
 
 // =========================
 // Start
 // =========================
-//x @step(r'I start the possibly error-producing interview "([^"]+)"')
+//x @step(r'I start the possibly error-producing interview "([^"]+)"') 
 //x @step(r'I am using the server "([^"]+)"')
 //x @step(r'I log in with "([^"]+)" and "([^"]+)"')
-//x @step(r'I launch the interview "([^"]+)"')
+//x @step(r'I launch the interview "([^"]+)"') 
 
 // ? @step(r'I start the interview "([^"]+)"')
 // ? I start the interview file "([^"]+)" on mobile
-// ? I open the interview file ?"?([^"]+)"?( on mobile/a phone/a computer/a pc/a tablet???)
+// ? I open the interview file ?"?([^"]+)"?( on mobile/a phone/a computer/a pc/a tablet???) 
 // ? I open the interview file ?"?([^"]+)"?[ on ]?(.*)/
-// √ I start the interview on mobile|pc
+// √ I start the interview on mobile|pc 
 
 
 // =========================
 // Observational/Passive
 // =========================
-//x @step(r'I wait forever')
+//x @step(r'I wait forever') 
 //x @step(r'I should see "([^"]+)" as the title of the page')
 //x @step(r'I should see "([^"]+)" as the URL of the page')
 //x @step(r'I save a screenshot to "([^"]+)"')
-//x @step(r'I set the window size to ([0-9]+)x([0-9]+)')
+//x @step(r'I set the window size to ([0-9]+)x([0-9]+)') 
 //x @step(r'I screenshot the page')
 
 // √ @step(r'I wait (\d+) seconds?')
 
-// Make writing tests more comfortable.
+// Make writing tests more comfortable. 
 // √ I do nothing
 
 // ~ I should see the term "([^"]+)"
 
-// √ /I (should|should not) see the phrase "([^"]+)"/i
+// √ /I (should|should not) see the phrase "([^"]+)"/i 
 
 // √ For things that have double quotes in them
-// √ I (should|should not) see the phrase '([^']+)'/i
+// √ I (should|should not) see the phrase '([^']+)'/i 
 
-// √ the question id should be {string}
+// √ the question id should be {string} 
 
 // √ I will be told an answer is invalid
 
-// ? @step(r'I should see that "([^"]+)" is "([^"]+)"')  // What do? Anything with label
+// ? @step(r'I should see that "([^"]+)" is "([^"]+)"')  // What do? Anything with label 
```

Comment:
This seems like a sentence with broad applications, similar to `I pick ""`. For example, this could cover the "I should see the title/URL of the page is ___" sentences.

Reply:
I think it's specifically referring to fields with labels. I thought it would leave it open to checkboxes, but `that "I accept" is "unchecked"` doesn't really make sense, so maybe it's not as open as I thought it was. That sentence would make sense for drop downs, comboboxes, radio buttons, text areas, and text fields, though, which is a lot.

Reply:
I gave it an 'x'.